pkg/builder/controller.go

```diff
@@ -445,9 +444,6 @@ func (blder *TypedBuilder[request]) doController(r reconcile.TypedReconciler[req
 			log := log
 
 			if req, ok := any(in).(*reconcile.Request); ok && req != nil {
-				if hasGVK {
-					log = log.WithValues(gvk.Kind, klog.KRef(req.Namespace, req.Name))
```

Comment:
The format of the two is different, this was added to be compatible with the format used by other projects in the ecosystem. Removing this would be a breaking change and I don't see a reason to do that. You can always set up your own logging and make the integrated a no-op.
/hold

Reply:
Agree. I would like to continue to align with the k/k log format